README.md

```diff
@@ -207,187 +250,226 @@ On first execution, this will generate golden snapshots. Subsequent runs will co
 npx folio --update-snapshots
 ```
 
------------
-
-## Configuration
-
-### Modify context options
+### Page object model
 
-You can modify the built-in fixtures. This example modifies the default `contextOptions` with a custom viewport size.
-
-**Step 1**: Create a new file (say `test/fixtures.ts`) which contains our modifications.
+To introduce a Page Object for a particular page, create a class that will use the `page` object.
 
+Create a `LoginPage` helper class to encapsulate common operations on the login page.
 ```ts
-// test/fixtures.ts
-import { folio as baseFolio } from "@playwright/test";
-
-const builder = baseFolio.extend();
+// login-page.ts
+export class LoginPage {
+  constructor(page) {
+    this.page = page;
+  }
 
-// Fixture modifications go here
+  async goto() {
+    await this.page.goto("https://example.com/login");
+  }
 
-const folio = builder.build();
+  async login() {
+    await this.page.fill("#username", TEST_USERNAME);
+    await this.page.fill("#password", TEST_PASSWORD);
+    await this.page.click("text=Login");
+  }
+}
 ```
 
-**Step 2**: Override the existing `contextOptions` fixture to configure viewport size and HTTPS error handling.
-
-```diff
-// test/fixtures.ts
-import { folio as baseFolio } from "@playwright/test";
-+ import { BrowserContextOptions } from "playwright";
-
-const builder = baseFolio.extend();
-
-+ builder.contextOptions.override(async ({ contextOptions }, runTest) => {
-+   const modifiedOptions: BrowserContextOptions = {
-+     ...contextOptions, // default options
-+     viewport: { width: 1440, height: 900 },
-+     ignoreHTTPSErrors: true
-+   }
-+   await runTest(modifiedOptions);
-+ });
-
-const folio = builder.build();
+Use the `LoginPage` class in the tests.
+```ts
+// my.spec.ts
+import { test, expect } from "@playwright/test";
```

Comment:
As a user who comes from other frameworks I commonly use `it` instead of test. Also sentence-wise it's more readable (e.g. it should do a when b is passed), can we export it as an alias or was this already clarified?

Reply:
Yeah, we can do an alias.